book/01-introduction/sections/about-version-control.asc

```diff
@@ -1,62 +1,62 @@
-=== About Version Control
+=== Tentang Version Control
 
 (((version control)))
-What is "version control", and why should you care?
-Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later.
-For the examples in this book you will use software source code as the files being version controlled, though in reality you can do this with nearly any type of file on a computer.
+Apakah "version control", dan mengapa sebaiknya Anda peduli?
+Version control adalah sebuah sistem yang merekam perubahan-perubahan kepada sebuah berkas atau sekumpulan berkas setiap waktu sehingga Anda dapat menilik kembali versi khusus suatu saat nanti.
+Sebagai contoh, pada buku ini Anda akan menggunakan sumber kode perangkat lunak sebagai berkas-berkas yang sedang dikenai _version control_, walau pada kenyataannya Anda dapat melakukan ini dengan hampir semua jenis berkas pada komputer.
```

Comment:
Itu bukan sebuah _typo,_ maksud saya memang dikenai. Saya kesulitan menemukan terjemahan yang pas untuk `as the files being version controlled` yang maksudnya adalah `yang mana file itu sedang di-version control`